payment/src/on_completed/main.py

```diff
@@ -76,3 +78,11 @@ def handler(event, _):
     payment_token = get_payment_token(order_id)
     process_payment(payment_token)
     delete_payment_token(order_id)
+
+    # Add custom metrics
+    amount_processed = event["detail"]["total"]
+
+    metrics.add_dimension(name="environment", value=ENVIRONMENT)
+    metrics.add_metric(name="paymentProcessed", unit=MetricUnit.Count, value=1)
+    metrics.add_metric(name="amountWin", unit=MetricUnit.Count, value=amount_processed)
```

Comment:
Small nit: the two metrics are `amountWin`/`amountLost`, it'd be better to have this as `amountWon` for coherence.